examples/module/src/gizmo/__init__.py

```diff
@@ -2,6 +2,7 @@
 This file registers the Gizmo subtype with the Viam Registry, as well as the specific MyGizmo model.
 """
 
+from viam.components.motor import *
```

Comment:
You should add an inline comment here explaining why we need this. Something like `from viam.components.motor import *  # Need to import motor to so the component registers itself`